deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -381,6 +381,8 @@ async def create_new_job(
             params.append("stiffness_factor")
         if pulse3d_semver >= "0.27.4":
             params.append("inverted_post_magnet_wells")
+        if pulse3d_semver >= "0.28.1":
+            params.append("include_stim_protocols")
```

Comment:
When we make changes to any of the services, we want to bump the version to tag the images.
the versions can be found in `/services/<namespace>/core/config.py` for all the services and then you'll want to bump the version tags in the manifests for the service as well. Including the prod overlay

Reply:
@luciipak I already added version 0.28.1 to the overlays  and just bumped the version in the config.py. Is that all I would need to do?

Reply:
the version in the config.py file is should be updated to 0.5.0?